packages/common/src/ui/components/domain/StatusCrumbs/StatusCrumbs.tsx

```diff
@@ -2,97 +2,72 @@ import React from 'react';
 import Breadcrumbs from '@mui/material/Breadcrumbs';
 import Typography from '@mui/material/Typography';
 import Box from '@mui/material/Box';
-
-import { useAppTheme } from '../../styles';
-import { ChevronDownIcon } from '../icons';
+import { ChevronDownIcon } from '../../../icons';
 import {
   LocaleKey,
   useTranslation,
   useFormatDate,
-} from '../../intl/intlHelpers';
-import { VerticalStepper } from './steppers/VerticalStepper';
-import { usePopover } from './popover';
+} from '../../../../intl/intlHelpers';
```

Comment:
just gone through and removed all the things that ended up being unneeded and added touchups like extracting the loop out to a hook